MatrixKit/Utils/MXKConstants.h

```diff
@@ -37,3 +37,15 @@ FOUNDATION_EXPORT NSString *const kMXKErrorNotification;
  */
 FOUNDATION_EXPORT NSString *const kMXKErrorUserIdKey;
 
+
+#pragma mark - Analytics
+
+/**
+ The analytics category for local contacts.
+ */
+FOUNDATION_EXPORT NSString *const kMXKAnalyticsContactsCategory;
```

Comment:
Prefix `k` convention is normally used for internal constants. As this is a new constant we can remove it.
We can also use a dedicated `MXKAnalyticsCategories.h` or `MXKAnalyticsConstants.h`.